### PR TITLE
Refactoring - separate Greek Paradigm

### PR DESCRIPTION
**Pull request by irina060981**
The step for the issue - https://github.com/alpheios-project/inflection-tables/issues/289

1. Create separate GreekParadigmDataset
2. Add ability to have more than one LanguageDataset for the languageID
3. Connect GreekParadigmDataset to specific Views
4. Place all Paradigm Specific code to a separate class
5. Update tests
6. Remove deprecated getInflectionData from LanguageDatasetFactory

### CODE REVIEW

File: views/lib/view.js [hunk 1]
```diff
@@ -336,7 +336,8 @@ export default class View {
 
   static getInflectionsData (homonym, options) {
     // Select inflections this view needs
-    return this.dataset.createInflectionSet(this.mainPartOfSpeech, this.getRelatedInflections(homonym.inflections), options)
+    const dataset = this.dataset
+    return dataset.createInflectionSet(this.mainPartOfSpeech, this.getRelatedInflections(homonym.inflections), options)
```
Comment (kirlat) [inline views/lib/view.js]: What is a difference with the old version? Maybe it's better to have it in one line?

File: views/lib/view.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ export default class View {
    * @return {LanguageDataset}
    */
   static get dataset () {
-    return LDF.getDataset(this.languageID)
+    return LDF.getDatasets(this.languageID)[0]
```
Comment (kirlat) [inline views/lib/view.js]: Maybe it's better to add a comment explaining why the first dataset only is returned and what this dataset is?

File: lib/language-dataset-factory.js [hunk 3]
```diff
@@ -36,35 +39,28 @@ export default class LanguageDatasetFactory {
...
-      return dataset.getInflectionData(homonym)
-    } else {
-      return new InflectionData(homonym) // Return an empty inflection data set
+  static getDataset (languageID, constructorName) {
+    const datasets = this.getDatasets(languageID)
+    if (!datasets) {
+      return
+    }
+    if (constructorName) {
+      return datasets.find(dataset => dataset.constructor.name.indexOf(constructorName) === (dataset.constructor.name.length - constructorName.length))
```
[the remark this answers is not in the document]
Comment (balmas) [reply lib/language-dataset-factory.js]: agree this test is a little unclear.

[the remark this answers is not in the document]
Comment (irina060981) [reply lib/language-dataset-factory.js]: This method `endsWith` is rather new from EcmaScript 6, didn't use it before.
Thanks for tellig about it :)
Will fix

File: views/lib/view.js (hunk 1 above)
Comment (irina060981) [reply views/lib/view.js]: I was sure - I fixed it!
Thank you for pointing to this.

File: views/lib/view.js (hunk 2 above)
Comment (irina060981) [reply views/lib/view.js]: Added

**Pull request merged**